pkg/cloud/rgraph/ez/ez.go

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package ez is a utility to create complex resource graphs for testing from a
+// concise description by use of naming conventions and default values.
+//
+// This package should only be used for testing. There is little error handling;
+// most errors will result in a panic() to reduce the verbosity of the code.
+package ez
```

Comment:
Can we update the package name to `eztesting` or `graphtesting` since not using this in prod requires people to read the docstring?

Reply:
maybe put it at `pkg/cloud/rgraph/testing/ez`?

Reply:
ok

Reply:
Is `testing` or `test`. one of these is somewhat special?

Reply:
I don't believe path the paths are special, but `testing` seems like a common directory or package name for test libraries / helpers.